generate/swaggergen/g_docs.go

```diff
@@ -356,12 +356,19 @@ func analyseControllerPkg(localName, pkgpath string) {
 	}
 	pkgRealpath := ""
 
-	wgopath := filepath.SplitList(gopath)
-	for _, wg := range wgopath {
-		wg, _ = filepath.EvalSymlinks(filepath.Join(wg, "src", pkgpath))
-		if utils.FileExists(wg) {
-			pkgRealpath = wg
-			break
+	wg, _ := filepath.EvalSymlinks(filepath.Join(vendorPath, pkgpath))
+	if utils.FileExists(wg) {
+		pkgRealpath = wg
+	} else {
+		wgopath := filepath.SplitList(gopath)
+		for _, wg := range wgopath {
+			beeLogger.Log.Infof("wg := range wgopath => %s", wg)
```

Comment:
Can you remove `beeLogger.Log.Infof()` or use `Debugf` with a clear debug message instead?